mirror/images.txt

```diff
@@ -9,12 +9,18 @@
 # ceph-csi devel
 docker.io/rook/ceph:v1.8.2	rook/ceph:v1.8.2
 
+
+# ceph-csi v3.5
+quay.io/cephcsi/ceph:v16	ceph/ceph:v16
```

Comment:
this looks incorrect, we do not use `quay.io/cephcsi/`, the images are located under `quay.io/ceph/`?